packages/google_maps_routes_api/lib/routes_service.dart

```diff
@@ -0,0 +1,127 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+library google_maps_routes_api;
+
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+
+import 'src/types/index.dart';
+
+/// A service used to calculate routes and route matrixes using
+/// Google Routes API REST endpoints.
+class RoutesService {
+  /// Creates the [RoutesService].
+  const RoutesService({required this.apiKey});
+
+  /// Google API key used for the requests.
+  final String apiKey;
+
+  static const String _routesApiUrl = 'https://routes.googleapis.com/';
+
+  /// POST https://routes.googleapis.com/directions/v2:computeRoutes
```

Comment:
To the first line add description of the function.
Then one empty comment line and current text.